cli/popper/utils.py

```diff
@@ -474,9 +474,30 @@ def get_name_and_path_for_new_pipeline(folder, pipeline_name=''):
     return new_pipeline_name, path
 
 
-def exec_cmd(cmd, ignoreerror=False):
+def exec_cmd_verbose(command):
```

Comment:
Given that this is similar to what's on `ActionRunner.execute`, we can take this opportunity to create a utility function for executing external commands with parameters to suppress output, only write output to `.out` and `.err` logs, or both.

Reply:
Does suppressing output mean printing ouput to STDOUT ?

Reply:
Sorry, I meant, four options:
  * suppress output (don't write output at all)
  * write to `stdout`
  * write to log files
  * both (write to `stdout` and log files)

Reply:
Thanks ! @ivotron